### PR TITLE
Remove 'sponsor' from Header on Partner page

### PR DESCRIPTION
**Pull request by brettski**

### CODE REVIEW

File: pages/wi/partners.js [hunk 1]
```diff
@@ -106,7 +106,11 @@ const partnerListing = () => {
       <ContentSection>
         <Grid columns="repeat(auto-fit,minmax(32rem,1fr))">
           <Cell>
-            <Header>2019 Sponsors & Partners</Header>
+            <Header>
+              2019
+              <br />
+              Partners
+            </Header>
```
Comment (saragibby) [inline pages/wi/partners.js]: When we need to wrap a header we will adjust the width and let it wrap naturally. Line breaks have a tendency to make this look off when screen sizes are adjusted so we avoid it as much as possible. I think in this case it is a matter of adjusting the graphic size, location, spacing, all that. 

We could add this as an open source issue, and I'm sure a bunch on contributors would happily pick it up and start playing with the graphic and responsiveness here.

Comment (brettski) [reply pages/wi/partners.js]: Yeah, the line break felt really wrong, so I figured one of you would set me straight!!

Comment (saragibby) [reply pages/wi/partners.js]: Haha, I know how tempting there are 😄 
For this I say just update it to "2019 Partners" and let's get it in. Good catch that we had "sponsors" still here. 

Then if the graphic is still off on the various size screens we open a Github issue for the community to pick up.

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/partners**](https://that-website-git-brettski-partner-page-rewording.that-conference.now.sh/wi/partners) |
|:-:|
|<a href="https://that-website-git-brettski-partner-page-rewording.that-conference.now.sh/wi/partners">Screenshot of /wi/partners</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/RyVmT92D24XK8PYQDyNM7bmbSAw_">(view full size)</a> |







Commit 597947e4da29fd0e21caaef0b510efa37bfbc1c1 (https://that-website-git-brettski-partner-page-rewording.that-conference.now.sh).

File: pages/wi/partners.js (hunk 1 above)
Comment (brettski) [reply pages/wi/partners.js]: Corrected Sara

**Pull request merged**